src/luarocks/repos.lua

```diff
@@ -347,7 +347,7 @@ local function op_rename(op)
    if fs.exists(op.src) then
       fs.make_dir(dir.dir_name(op.dst))
       fs.delete(op.dst)
-      local ok, err = fs.move(op.src, op.dst)
+      local ok, err = os.rename(op.src, op.dst)
```

Comment:
Could this one end up trying to rename across file systems?

Reply:
no, this is used to really _rename_ things within a LuaRocks tree, and not move files around. For actual deployment of files ,`fs.move` is still used to move files (by copying and deleting them) one by one.